news/13346.bugfix.rst

```diff
@@ -0,0 +1 @@
+This change will improve error hint for long filenames on Windows (EINVAL)
```

Comment:
```
Provide a hint if a system error is raised involving long filenames or path segments on Windows. 
```

The changelog entry should be worded for the user. No need for "this change will [...]" as that implied.